src/organisms/cards/SelectCard/select-card.module.scss

```diff
@@ -9,20 +9,27 @@ $footer-link-hover-color:  color('neutral-2');
 
 .select {
   composes: card;
+
   border: $select-card-border;
   border-bottom-width: ru(.125);
 }
 
+// Selectfield needs to be stylized individually when in a cardText
+// label -> field (ru(.25) top & bottom)
```

Comment:
Why this comment? I don't really understand what it's saying. Maybe expound a bit?